movingpandas/trajectory.py

```diff
@@ -879,6 +879,37 @@ def get_sampling_interval(self):
                 return self.df[self.timedelta_col_name].median()
         return self._get_df_with_timedelta()[TIMEDELTA_COL_NAME].median()
 
+    def _get_global_velocity_length(self, units=UNITS()):
+        pt_tuples = [(pt.y, pt.x) for pt in [self.get_start_location(),
+                                             self.get_end_location()]]
+        if self.is_latlon:
+            length = geodesic(*pt_tuples).m
```

Comment:
Please use the existing geometry_utils._measure_distance function  https://github.com/movingpandas/movingpandas/blob/39d5b125ec1b148bb0087b257178f9e631b7c67b/movingpandas/geometry_utils.py#L84